Svc/SystemResources/SystemResources.fpp

```diff
@@ -129,12 +109,6 @@ module Svc {
     @ System's CPU Percentage
     telemetry CPU_15: F32 id 20 format "{.2f} percent"
 
-    @ Software framework version
-    telemetry FRAMEWORK_VERSION: string size 40 id 21
-
-    @ Software project version
-    telemetry PROJECT_VERSION: string size 40 id 22
```

Comment:
This requires changes to the tutorial reference repositories because they have the Telemetry spec XML file... that is why CI is breaking in some places.

Which begs the question, should that file be removed from the tutorials? It's really not that useful...
I vote yes 👀 

@kevin-f-ortega thoughts? Since you are reworking the LedBlinker one

Reply:
Looks like the xml file in Ref/Top needed updating as well. That took care of some of these warnings